style.css

```diff
@@ -0,0 +1,10 @@
+body{
+    padding: 0;
+    margin: 0;
+  }
+  #main{
+    background: silver;
+    text-align: center;
+    
+  }
```

Comment:
Spacing required between css selectors and the opening brace. please kindly add linters to your project. linters make your code look clean and professional